docs/client-development/mandate.rst

```diff
@@ -0,0 +1,520 @@
+    Content-Type: application/json
+
+    {
+        "zaak": "http://example.com",
+        "betrokkeneType": "niet_natuurlijk_persoon",
+        "roltype": "http://example.com/roltype-belanghebbende",
+        "roltoelichting": "Bedrijf dat de intermediair machtigt",
+        "indicatieMachtiging": "machtiginggever",
+        "betrokkeneIdentificatie": {
+            "kvkNummer": "99998888"
+        },
+        "authenticatieContext": null
+    }
+
+
+Query patterns
+--------------
+
+Below are examples how to request zaken, authorized by different parties.
+
+**DigiD**
+
+1. Show me my cases (based on my BSN) opened for myself:
+
+    .. code::
+
+        POST /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__natuurlijkPersoon__inpBsn=<ownBsn>
+             &rol__machtiging=eigen
+
+
+    Additionally ``rol.omschrijvingGeneriek`` can be used to determine the "initiator"
+    of the case.
+
+    .. code::
+
+        POST /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__natuurlijkPersoon__inpBsn=<ownBsn>
+             &rol__machtiging=eigen
+             &rol__omschrijvingGeneriek=initiator
+
+
+2. Show me my cases (based on my BSN) opened by an authorizee on my behalf:
+
+    .. code::
+
+        POST /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__natuurlijkPersoon__inpBsn=<ownBsn>
+             &rol__machtiging=machtiginggever
+
+
+    Additionally ``rol.omschrijvingGeneriek`` can be used to determine the "belanghebbende"
+    of the case.
+
+    .. code::
+
+        POST /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__natuurlijkPersoon__inpBsn=<ownBsn>
+             &rol__omschrijvingGeneriek=belanghebbende
+
+
+3. Show me the cases (based on my BSN) where I represent another party:
+
+    .. code::
+
+        POST /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__natuurlijkPersoon__inpBsn=<ownBsn>
+             &rol__machtiging=gemachtigde
+
+
+    It's also possible to filter cases based on the level of assurance and to exclude
+    results where levelOfAssurance is below some required value.
+
+    .. code::
+
+        POST /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__natuurlijkPersoon__inpBsn=<ownBsn>
+             &rol__machtiging=gemachtigde
+             &rol__machtiging__loa=urn:oasis:names:tc:SAML:2.0:ac:classes:MobileTwoFactorContract
+
+
+**eHerkenning**
+
+Filters for eHerkenning authorized parties depend if it's an organization or the branch, therefore
+there are two examples for each option.
+
+1. Show me my cases (based on my KVK nummer) opened for myself:
+
+    .. code::
+
+        # organization
+        POST /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__nietNatuurlijkPersoon__kvk_Nummer=<ownKvk>
+             &rol__machtiging=eigen
+
+        # branch
+        POST /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__vestiging__kvkNummer=<ownKvk>
+             &rol__machtiging=eigen
```

Comment:
Should these be separate code blocks and also of type HTML?

Reply:
Before this code block it's said that there are two examples (each for different rol type)
I can separate them into different blocks, do you think it will increase readability? 

Please take a look at my comment above about "http" for "code" directive. Sphinx keeps raising validation errors if I put it here and I can't resolve it

Reply:
![Screenshot from 2024-10-30 10-56-57](https://github.com/user-attachments/assets/8e9bd011-d12f-4752-8f72-adf31f63cb45)

It looks slightly better I guess. Not really needed.